Single functions to get/set every celestial vault element (#12181)
minetest/minetest: doc/lua_api.txt

          nodes square)
    * `selected_mode` is the mode index to be selected after modes have been changed
    (0 is the first mode).
* `set_celestial_vault(vault_parameters)`
    *  Allows to set every celestial vault element in one take.
    * `vault_parameters` is a table optionally containing the tables `sky`, `sun`,
      `moon`, `stars` and `clouds`. These tables take in the parameters
      indicated in their more specific functions (`set_sky()`, `set_sun()` etc).
   * Passing no arguments resets every element of the vault to its default values.
     To reset just a single element, its specific function must be used instead.
* `get_celestial_vault()`

>     * returns a table containing every celestial vault element of the player, in
>       the format `sky`, `sun`, `moon`, `stars` and `clouds` as tables. See their
>       corrispondent specific functions to know what each one of them contains (`get_sky()`, 
>       `get_sun()` etc).

Reviewer: ```suggestion
    * returns a table containing every celestial vault element of the player, with
      `sky`, `sun`, `moon`, `stars` and `clouds` as tables. See their correspondent
       functions to know what each one of them contains (`get_sky()`, `get_sun()` etc).
```
